Makefile.am

```diff
@@ -10,7 +10,6 @@ EXTRA_DIST = \
 	autogen.sh \
 	harfbuzz.doap \
 	README.md \
-	README.mingw.md \
```

Comment:
Why is this?

Reply:
It is really not something intended for the release package.